src/System.IO.MemoryMappedFiles/tests/MemoryMappedFile.CreateFromFile.Tests.cs

```diff
@@ -349,40 +359,30 @@ public void InvalidArguments_Inheritability(HandleInheritability inheritability)
         }
 
         /// <summary>
-        /// Test various combinations of arguments to CreateFromFile, focusing on the Create and Truncate modes,
+        /// Test various combinations of arguments to CreateFromFile, focusing on the Create mode,
         /// and validating the creating maps each time they're created.
         /// </summary>
         [Theory]
-        [MemberData(nameof(MemberData_ValidArgumentCombinationsWithPath),
-            new FileMode[] { FileMode.Create, FileMode.Truncate },
-            new string[] { null, "CreateUniqueMapName()" },
-            new long[] { 1, 256, -1 /*pagesize*/, 10000 },
-            new MemoryMappedFileAccess[] { MemoryMappedFileAccess.Read, MemoryMappedFileAccess.ReadWrite, MemoryMappedFileAccess.CopyOnWrite })]
-        public void ValidArgumentCombinationsWithPath_ModesCreateOrTruncate(
-            FileMode mode, string mapName, long capacity, MemoryMappedFileAccess access)
+        [InlineData(1)]
+        [InlineData(256)]
+        [InlineData(10000)]
+        public void ValidArgumentCombinationsWithPath_ModeCreate(long capacity)
```

Comment:
I got confused because the MemberData helper method was filtering the inputs to netcoreapp-valid combinations which in and of itself is fine, but since I was altering what was "valid" I thought I was getting coverage where actually the inputs were being filtered based on the old behavior.

For example, the only MemoryMappedFileAccess from the above 3 compatible with FileMode.Create is `ReadWrite`, so the other two MMFA's being passed to the helper were getting filtered out for all combinations. The named-map was also being filtered out since I was testing on Linux. Since I narrowed the FileModes, mapNames, and MMFAs down to one item each, it seemed unnecessary to use the MemberData helper instead of just passing the capacities as InlineData directly.

I am missing the pagesize argument and mapName testing on Windows, so it is a loss of coverage either way. I'll refactor things a bit to get that coverage back.